src/Makefile.am

```diff
@@ -160,6 +160,23 @@ pkginclude_HEADERS += $(HB_SUBSET_headers)
 pkgconfig_DATA += harfbuzz-subset.pc
 EXTRA_DIST += harfbuzz-subset.pc.in
 
+harfbuzz-subset.cc: Makefile.sources
+	$(AM_V_GEN) \
+	for f in \
+		$(HB_BASE_sources) \
+		$(HB_GLIB_sources) \
+		$(HB_FT_sources) \
+		$(HB_GRAPHITE2_sources) \
+		$(HB_UNISCRIBE_sources) \
+		$(HB_GDI_sources) \
+		$(HB_DIRECTWRITE_sources) \
+		$(HB_CORETEXT_sources) \
```

Comment:
You *just* need `HB_SUBSET_sources` here to build harfbuzz-subset library.

Reply:
The idea is to build a (static) standalone hb-subset, but may be that is too WASM-specific. I guess one can always `c++ harfbuzz.cc harfbuzz-subset.cc` to achieve the same.

Reply:
> I guess one can always c++ harfbuzz.cc harfbuzz-subset.cc to achieve the same.

Yes, if that works, I prefer that. Then `harfbuzz-subset.cc` will be equivalent to `libharfbuzz-subset`.

Reply:
Okay, then at least just include the base HB sources. No need for any auxiliary sources; unless you need those as well.

Reply:
Done.